src/bundler.ts

```diff
@@ -191,14 +218,20 @@ class Bundler {
 
     const imprt = importMap.get(resolvedUrl);
     if (imprt) {
+      // If the document wasn't loaded for the import during analysis, we can't
+      // inline it.
+      if (!imprt.document) {
+        // TODO(usergenic): What should the behavior be when we don't have the
+        // document to inline available in the analyzer?
+        return;
```

Comment:
I don't know, since you can develop without bundling, I'm not sure that a file load shouldn't be a fatal error.